### PR TITLE
Retinas crop tool

### PR DESCRIPTION
**Pull request by eatyourgreens**
@brian-c @camallen @aliburchard 
This adds 'crop', as a rectangle tool option, to the drawing tools dropdown, then changes the viewBox if a crop rectangle annotation exists when you start a new task.

Integration with the classifier is still a big hack, but this might be good enough to get the Retinas project going on PFE.

### CODE REVIEW

**Comment by brian-c** [comment]
Nicely isolated. Can we block the tool from the [list in the editor](https://github.com/zooniverse/Panoptes-Front-End/blob/retinas/app/classifier/tasks/generic-editor.cjsx#L95-L96), maybe showing it [if the project owner is an admin](https://github.com/zooniverse/Panoptes-Front-End/blob/retinas/app/pages/lab/workflow.cjsx#L108-L119)?

**Comment by eatyourgreens** [comment]
I've made a start on checking the owner. Looks like staging isn't returning `owner.admin` for Ali though. I'll have another look at this tomorrow.

File: app/classifier/tasks/generic-editor.cjsx [hunk 1]
```diff
@@ -7,6 +7,34 @@ alert = require '../../lib/alert'
 DrawingTaskDetailsEditor = require './drawing-task-details-editor'
 NextTaskSelector = require './next-task-selector'
 
+DrawingToolsEditor = React.createClass
```
Comment (brian-c) [inline app/classifier/tasks/generic-editor.cjsx]: Why's this piece its own module? Can't we just just show or hide the `<option>`?

If it's got to be its own module, can we include all the drawing task stuff?

File: app/classifier/tasks/generic-editor.cjsx [hunk 2]
```diff
@@ -88,15 +116,7 @@ module.exports = React.createClass
...
-                  [<div key="type" className="workflow-choice-setting">
-                    <AutoSave resource={@props.workflow}>
-                      Type{' '}
-                      <select name="#{@props.taskPrefix}.#{choicesKey}.#{index}.type" value={choice.type} onChange={handleChange}>
-                        {for toolKey of drawingTools
-                          <option key={toolKey} value={toolKey}>{toolKey}</option>}
-                      </select>
-                    </AutoSave>
-                  </div>
+                  [<DrawingToolsEditor project={@props.project} workflow={@props.workflow} name="#{@props.taskPrefix}.#{choicesKey}.#{index}.type" value={choice.type} onChange={handleChange} />
```
Comment (brian-c) [inline app/classifier/tasks/generic-editor.cjsx]: Components in arrays need a `key` prop.

**Pull request closed without merging**